src/test/java/com/faforever/api/avatar/AvatarServiceTest.java

```diff
@@ -0,0 +1,151 @@
+package com.faforever.api.avatar;
+
+import com.faforever.api.config.FafApiProperties;
+import com.faforever.api.data.domain.Avatar;
+import com.faforever.api.data.domain.AvatarAssignment;
+import com.faforever.api.error.ApiExceptionWithCode;
+import com.faforever.api.error.ErrorCode;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.junit.rules.TemporaryFolder;
+import org.junit.runner.RunWith;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import javax.persistence.EntityNotFoundException;
+import java.io.InputStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Collections;
+import java.util.Optional;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@RunWith(MockitoJUnitRunner.class)
+public class AvatarServiceTest {
+
+  private static final long VALID_FILE_SIZE = 1024;
+  private static final long TOO_BIG_FILE_SIZE = 2048;
+  private static final String TOOLTIP = "Very nice avatar";
+  private static final String VALID_AVATAR_FILENAME = "qai2.png";
+  private static final String BIG_AVATAR_FILENAME = "donator.png";
+  private static final String LONG_AVATAR_FILENAME = "CachedAvataravatar_FightNight_Champion.png";
+  private static final String INVALID_EXTENSION_AVATAR_FILENAME = "supcom.jpg";
+  protected static final int AVATAR_ID = 1;
```

Comment:
No reason for `protected`?

Reply:
yep sry for some reason refactor extract constant used protected